collection-scripts/targeted_crs

```diff
@@ -135,6 +135,13 @@ if [ ! -z "${vm_resources}" ]; then
         dump_resource "virtualmachine" $target_vm_name $target_ns
         present_vm_resources+=("$target_vm_name")
 
+        if [ -z $PLAN ]; then
```

Comment:
what's the assumption here? (if a plan is not specified then .. )

Reply:
then, the above migration from the plan didn't set. and you must have either plan or vm+namespace.

Reply:
I think we wouldn't fail because the targeted scripts would probably not be called or conditions like `if [ -z $NS ] && [ -z $PLAN ] && [ -z $VM ]` would return false - but need to check
as for the flows you listed above, there seems to be an additional one in which you provide a namespace - need to support that as well or to drop it (but only after we verify it's really not needed)

Reply:
> if it's migration then you shold have the vm and namespace

actually you raise a good point here - if we want to work with migration here, why do we still have the option of namespace / plan / vm and don't have one for migration? I would expect the UX to trigger gathering for a plan rather than for a vm and that would render some of the changes in this PR redundant, no?

Reply:
> uld expect the UX to trigger gathering for a migration rather than for a vm and that would render some of the changes in this PR redundant, no?

this works. it gathers everything from the cluster that relates to vms/forklift.